tao_compiler/mlir/disc/transforms/disc_pdl_utils.cc

```diff
@@ -227,6 +228,24 @@ static LogicalResult checkConstantTensor(PatternRewriter& rewriter,
   return matchPattern(v, m_Constant(&denseAttr)) ? success() : failure();
 }
 
+static LogicalResult checkConstantTensorValueIsOne(PatternRewriter& rewriter,
```

Comment:
better to use Attribute as const value input？ e.g. CheckTensorValueIsOne(val, attr<"1">)?

Reply:
done